setup.py

```diff
@@ -1,11 +1,26 @@
+import sys
+
 from setuptools import setup
 
-long_description = open("README.rst").read()
+
+def read_file(fname):
+    with open(fname) as f:
+        return f.read()
+
+
+requirements = [
+        'pytest==3.1.2',
+        'requests==2.18.1',
+        'simplejson==3.11.1',
+]
+if sys.version_info.major == 2:
+    requirements.append('configparser==3.5.0')
+
 
 setup(
     name='pytest-testrail',
     description='pytest plugin for creating TestRail runs and adding results',
-    long_description=long_description,
+    long_description=read_file('README.rst'),
     version='0.0.11',
```

Comment:
Suggest bumping this to v1, since there are major changes and improvements

Reply:
Done